xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd

```diff
@@ -952,19 +920,37 @@ Rail transport, Roads and Road transport
 	<!-- ======================================================================= -->
 	<xsd:complexType name="linesInDirectionRefs_RelStructure">
 		<xsd:annotation>
-			<xsd:documentation>Type for a list of LINE in a specific DIRECTION</xsd:documentation>
+			<xsd:documentation>Type for a list of LINEs in a specific DIRECTION</xsd:documentation>
 		</xsd:annotation>
 		<xsd:complexContent>
 			<xsd:extension base="containmentAggregationStructure">
-				<xsd:sequence>
-					<xsd:element ref="LineInDirectionRef" maxOccurs="unbounded"/>
-				</xsd:sequence>
+				<xsd:choice maxOccurs="unbounded">
+					<xsd:element ref="LineInDirectionRef">
+						<xsd:annotation>
+							<xsd:documentation>Reference to a LINE in a specifiic DIRECTION</xsd:documentation>
+						</xsd:annotation>
+					</xsd:element>
+					<xsd:element ref="AllowedLineDirectionRef"/>
+				</xsd:choice>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:complexType name="linesInDirections_RelStructure">
```

Comment:
Where is this _linesInDirections_RelStructure_ type used ? I cant find it

Reply:
@Aurige remove it? Or use it?

Reply:
@ue71603 remove if not used